src/record/record-transition.js

```diff
@@ -240,17 +304,18 @@ RecordTransition.prototype._next = function () {
     this._record._d = this._currentStep.data
   }
 
-  /*
-   * Please note: saving to storage is called first to allow for synchronous cache
-   * responses to destroy the transition, it is however not on the critical path
-   * and the transition will continue straight away, rather than wait for the storage response
-   * to be returned.
-   */
+  this._cacheResponses++
+	/*
```

Comment:
update comment to mention that if write acks exist transition will only get destroyed on storage response?